vcpkg/consume/install-locally-modified-package.md

```diff
@@ -0,0 +1,251 @@
+---
+title: "Tutorial: Install a locally modified dependency"
+description: This tutorial shows you how to modify a vcpkg port and install it using overlays"
+author: vicroms
+ms.author: viromer
+ms.date: 10/24/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+
+#CustomerIntent: As a vcpkg user, I want to learn how to modify a vcpkg provided dependency and install it
+---
+
+# Tutorial: Install a locally modified dependency
+This tutorial guides you on how to modified a vcpkg port using custom overlays.
```

Comment:
```suggestion
This tutorial guides you on modifying a vcpkg port using custom overlays.
```

Reply:
Also maybe add a sentence telling the reader what they will get out of following this tutorial. How can overlays help them? When should they use them?